src/helpers.ts

```diff
@@ -309,15 +309,15 @@ export const combineSDJWT = (jwt: string, disclosures: string[], kbjwt?: string)
   return combined;
 };
 
-export const createDecoy = (count: number, hasher: Hasher): string[] => {
+export const createDecoy = (count: number, hasher: Hasher, saltGenerator: SaltGenerator = generateSalt): string[] => {
```

Comment:
@ragnika if `saltGenerator` is required we don't need to add any default values.
If we keep this logic - please write a test or two to cover this part.